x-pack/legacy/plugins/apm/public/components/app/ServiceDetails/ServiceIntegrations/WatcherFlyout.tsx

```diff
@@ -260,7 +260,8 @@ export class WatcherFlyout extends Component<
             }
           )}{' '}
           <KibanaLink
-            path={`/management/elasticsearch/watcher/watches/watch/${id}`}
+            path={`/management/elasticsearch/watcher/watches/watch/${id}/status`}
```

Comment:
Did the endpoint change?

Reply:
@sqren, I changed it because the old link was going to the Watcher List Page instead of opening the watcher which has just been created. Unless that is the correct behavior?

Reply:
This looks good. Thanks for catching this 👍